Content.Server/NPC/Components/NPCRangedCombatComponent.cs

```diff
@@ -40,6 +40,12 @@ public sealed partial class NPCRangedCombatComponent : Component
     [ViewVariables(VVAccess.ReadWrite)]
     public bool TargetInLOS = false;
 
+    /// <summary>
+    /// If true, only opaque objects will block line of sight.
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite)]
+    public bool UseOpaqueForLOSChecks = false;
```

Comment:
The name throws a fit with our field naming conventions. Should probably shut up rider by adding
```suggestion
	// ReSharper disable once InconsistentNaming
    public bool UseOpaqueForLOSChecks = false;
```